modules/auth/site/templates/block_account_info.tpl

```diff
@@ -0,0 +1,60 @@
+ 	<div class="primary-tabs">
```

Comment:
just for the record
template    vs     cms_form:

template provides:
- a way to separate view from controller
- easy to edit to customize a site

cms form:
- provides a way for module to alter the form tree.
- provides a way to use translated text ..
- may be faster than loading template ... but make executable bigger.

for now, this is ok to use template ... but in the future we need to think twice about it ...
how it will be used, if a module may need to alter the content ... and so on.